lib/remote_files/fog_store.rb

```diff
@@ -98,5 +95,28 @@ def create_directory
         dir.save
       end
     end
+
+    def store_options(file)
+      store_options =
+        {
+          :body => file.content,
+          :content_type => file.content_type,
+          :key => file.identifier,
+          :public => options[:public],
+          :encryption => options[:encryption]
+        }
+      if file.options[:multipart_chunk_size]
```

Comment:
I should probably check that this relates to the AWS provider in this conditional.